app/services/stake_pools/calculate_average_stats.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module StakePools
+  class CalculateAverageStats
+    def initialize(pool)
+      @pool = pool
+    end
+
+    def call
+      @sp_attrs = initial_attrs
+
+      Validator.where(id: validator_ids)
+               .includes(:stake_accounts, :validator_score_v1)
+               .find_each do |validator|
+        @validator = validator
```

Comment:
I don't think this is used anywhere

Reply:
I've added a getter in terms of validator in line 34.